src/components/Homepage/Changelog.tsx

```diff
@@ -0,0 +1,188 @@
+import { useStaticQuery } from 'gatsby';
+import { graphql } from 'gatsby';
+import { Key } from 'react';
+import Badge from '@ably/ui/core/Badge';
+import FeaturedLink from '@ably/ui/core/FeaturedLink';
+import Link from '../Link';
+
+interface ChangelogFeedItemNode {
+  title: string;
+  link: string;
+  content: string;
+  isoDate: string;
+}
+
+interface ChangelogFeedEdge {
+  node: ChangelogFeedItemNode;
+}
+
+interface ChangelogQueryData {
+  allFeedAblyChangelog: {
+    edges: ChangelogFeedEdge[];
+  };
+  feedAblyChangelogMeta: {
+    link: string;
+  };
+}
+
+const ChangelogEntry = ({
+  date,
+  href,
+  title,
+  description,
+  tags,
+}: {
+  date: string;
+  href: string;
+  title: string;
+  description: string;
+  tags?: string[];
+}) => {
+  const getTagStyle = (tag: string) => {
+    const lowerTag = tag.toLowerCase();
+
+    switch (true) {
```

Comment:
is it more readable to use if condition here?

```
const getTagStyle = (tag: string) => {
  const tagLower = tag.toLowerCase();

  if (tagLower.includes('fix')) return 'blue';
  if (tagLower.includes('improvement')) return 'green';
  if (tagLower.includes('new')) return 'red';
  return 'neutral';
};
```

Reply:
Definitely, `switch (true)` is rogue